info/kubelessInfo.js

```diff
@@ -0,0 +1,152 @@
+      unsupportedOptions,
+      this.options,
+      this.serverless.cli.log.bind(this.serverless.cli)
+    );
+    return BbPromise.resolve();
+  }
+
+  formatMessage(service, functions, options) {
+    if (options && !options.color) chalk.enabled = false;
+    let message = '';
+    message += `\n${chalk.yellow.underline('Service Information')}\n`;
+    message += `${chalk.yellow('Service: ')} ${service.name}\n`;
+    message += `${chalk.yellow('Cluster IP: ')} ${service.ip}\n`;
+    message += `${chalk.yellow('Ports: ')}\n`;
+    _.each(service.ports, (port) => {
+      // Ports can have variable properties
+      _.each(port, (value, key) => {
+        message += `  ${chalk.yellow(`${toMultipleWords(key)}: `)} ${value}\n`;
+      });
+    });
+    message += `${chalk.yellow('Functions: ')}\n`;
+    _.each(functions, f => {
+      message += `  ${chalk.yellow(`${f.name}:`)}\n`;
+      message += `    ${chalk.yellow('Handler: ')} ${f.handler}\n`;
+      message += `    ${chalk.yellow('Runtime: ')} ${f.runtime}\n`;
+      message += `    ${chalk.yellow('Topic: ')} ${f.topic}\n`;
+      message += `    ${chalk.yellow('Dependencies: ')} ${f.deps}\n`;
+    });
+    if (this.options.verbose) {
+      message += `\n${chalk.yellow.underline('Service Metadata')}\n`;
+      message += `${chalk.yellow('Self Link: ')} ${service.selfLink}\n`;
+      message += `${chalk.yellow('UID: ')} ${service.uid}\n`;
+      message += `${chalk.yellow('Timestamp: ')} ${service.timestamp}\n`;
+      message += `${chalk.yellow('Functions Metadata:')}\n`;
+      _.each(functions, f => {
+        message += `  ${chalk.yellow(`${f.name}:`)}\n`;
+        message += `    ${chalk.yellow('Self Link: ')} ${f.selfLink}\n`;
+        message += `    ${chalk.yellow('UID: ')} ${f.uid}\n`;
+        message += `    ${chalk.yellow('Timestamp: ')} ${f.timestamp}\n`;
+      });
+    }
+    return message;
+  }
+
+  infoFunction(options) {
+    const core = new Api.Core(
+      Object.assign(helpers.getMinikubeCredentials(), {
+        url: process.env.KUBE_API_URL,
+        group: 'k8s.io',
+      })
+    );
+    const thirdPartyResources = new Api.ThirdPartyResources(
+      Object.assign(helpers.getMinikubeCredentials(), {
+        url: process.env.KUBE_API_URL,
+        group: 'k8s.io',
+      })
+    );
+    thirdPartyResources.addResource('functions');
+    const func = this.serverless.service.service;
+    return new BbPromise((resolve) => {
+      core.services.get(
+        { qs: { labelSelector: `function=${func}` } },
+        (err, info) => {
+          if (_.isEmpty(info.items)) {
+            throw new this.serverless.classes.Error(
+              `Unable to find the service for the function ${func}`
+            );
+          } else if (info.items.length > 1) {
+            throw new this.serverless.classes.Error(
+              `Found more than one service for the function ${func}`
+            );
+          }
+          const service = {
```

Comment:
do you think it makes sense to also show the service type?

`type: info.items[0].spec.type,`

If we plan to support different providers apart from Minikube I think it would be useful